cmd/globals.go

```diff
@@ -89,17 +90,23 @@ var (
 	// Terminal height/width, zero if not found
 	globalTermWidth, globalTermHeight int
 
-	globalDisablePagerFlag = "--disable-pager"
-	globalPagerDisabled    = false
-	globalHelpPager        *termPager
+	globalDisablePagerEnv       = "DISABLE_PAGER"
+	globalDisablePagerFlag      = "--disable-pager"
+	globalDisablePagerFlagShort = "--dp"
+	globalPagerDisabled         = false
+	globalHelpPager             *termPager
 
 	// CA root certificates, a nil value means system certs pool will be used
 	globalRootCAs *x509.CertPool
 )
 
 func parsePagerDisableFlag(args []string) {
+	disableEnv := os.Getenv(envPrefix + globalDisablePagerEnv)
+	if disableEnv == "true" {
+		globalPagerDisabled = true
+	}
```

Comment:
Simpler/Better?

```suggestion
	globalPagerDisabled, _ = strconv.ParseBool(os.Getenv(envPrefix + globalDisablePagerEnv))
```

I don't see it being written to elsewhere, so this seems good to me. 

ParseBool has a bit more options... and this matches other bool parsing AFAICT:

```
func ParseBool(str string) (bool, error) {
	switch str {
	case "1", "t", "T", "true", "TRUE", "True":
		return true, nil
	case "0", "f", "F", "false", "FALSE", "False":
		return false, nil
	}
	return false, syntaxError("ParseBool", str)
}
```